common-lib/project.clj

```diff
@@ -41,17 +41,17 @@
                  [org.clojure/tools.nrepl "0.2.13"]
                  [org.clojure/tools.reader "1.3.2"]
                  ;;these dependencies should be updated in tandem with the ring dependiences below
-                 [org.eclipse.jetty/jetty-http "9.4.39.v20210325"]
-                 [org.eclipse.jetty/jetty-io "9.4.39.v20210325"]
-                 [org.eclipse.jetty/jetty-servlets "9.4.39.v20210325"]
-                 [org.eclipse.jetty/jetty-util "9.4.39.v20210325"]
+                 [org.eclipse.jetty/jetty-http "9.4.48.v20220622"]
+                 [org.eclipse.jetty/jetty-io "9.4.48.v20220622"]
+                 [org.eclipse.jetty/jetty-servlets "9.4.48.v20220622"]
+                 [org.eclipse.jetty/jetty-util "9.4.48.v20220622"]
                  ;; load jts core lib first to make sure it is available for shapefile integration,
                  ;; otherwise ES referenced 1.15.0 version will be mistakenly picked for shapefile
                  [org.locationtech.jts/jts-core "1.18.2"]
                  [org.ow2.asm/asm "7.0"]
                  [potemkin "0.4.5"]
-                 [ring/ring-core "1.9.2"]
-                 [ring/ring-jetty-adapter "1.9.2"]
+                 [ring/ring-core "1.9.6"]
```

Comment:
I think we'll want to synch all of the ring upgrades. Ring is also being imported by `system-int-test/project.clj`, 'search-app/project.clj', 'metadata-db-app/project.clj'